tests/test-speech.yaml

```diff
@@ -332,4 +332,164 @@
               python main.py
             fi
         input: |-
-          Hello world.
\ No newline at end of file
+          Hello world.
+
+- area: ai speech synthesize
+  tests:
+  - name: Hello world
+    command: ai speech synthesize --text "Hello world" --audio-output hello-world.wav
+    parallelize: true
+    expect-regex: |
+      SYNTHESIS STARTED:
+      SYNTHESIS COMPLETED:
+
+- area: ai speech transcribe
+  tests:
+  - name: with file
+    command: ai speech transcribe --file ../tests/audio/hello.wav
+    parallelize: true
+    expect-regex: |
+      Hello.
+
+  - name: with multiple files
+    command: ai speech transcribe --files ../tests/audio/hello.wav;../tests/audio/hello.wav
+    parallelize: true
+    expect-regex: |
+      Hello.
+      Hello.
+
+  - name: with url
+    command: ai speech transcribe --url https://crbn.us/hello.wav
+    parallelize: true
+    expect-regex: |
+      Hello.
+
+  - name: with multiple urls
+    command: ai speech transcribe --urls https://crbn.us/hello.wav;https://crbn.us/hello.wav
+    parallelize: true
+    expect-regex: |
+      Hello.
+      Hello.
+
+  - name: with multiple files (threads 2)
+    command: ai speech transcribe --files ../tests/audio/hello.wav;../tests/audio/hello.wav --threads 2
+    parallelize: true
+    expect-regex: |
+      Hello.
+      Hello.
+
+  - name: with multiple files (processes 2)
+    command: ai speech transcribe --files ../tests/audio/hello.wav;../tests/audio/hello.wav --processes 2
+    parallelize: true
+    expect-regex: |
+      Hello.
+      Hello.
+
+  - name: without diarization
+    parallelize: true
+    command: ai speech transcribe --file katiesteve_mono.wav --input-path ../tests/audio
+    expect-regex: |
+      Good morning, Steve\. Good morning, Katie\.
+
+  - name: with diarization
+    command: ai speech transcribe --file katiesteve_mono.wav --input-path ../tests/audio --diarization --max-speakers 2
+    parallelize: true
+    expect-regex: |
+      Speaker 1: Good morning, Steve.
+      Speaker 2: Good morning, Katie.
```

Comment:
Does the diaziration get to the output files also?

Reply:
you can output the json from the service, or you can output the text in the "style" that the CLI already hs... e.g.

`--output-json-file response.json` will create a file called response.json, that will have the full json payload from the servcie.

or

`--output-file OUTPUT` -- will put output in that file (could be json, or tsv, it's tsv by default... just cause it is from the beginning of time w/ SPX)
`--output-text` will ensure text is output to that... it will either be a big "blob" of text, with spaces separating phrases, or, if you'e turned on diarization, it'll have speaker N: blah format... 
`--output-file-type json` will make it be json
`--output-file-type tsv` will make it be tsv

see an example at `output result text` test...

Reply:
If I go look in transcribe.output.examples

There's lots of examples of things I could put on a command line. But not a lot of content about what those things will get me.

That test validates there's a file, but not what's in the file.